Source/Scene/Cesium3DTile.js

```diff
@@ -383,6 +383,21 @@ define([
             }
         },
 
+        /**
+         * Get the tile's bounding volume.
+         *
+         * @memberof Cesium3DTile.prototype
+         *
+         * @type {TileBoundingVolume}
+         * @readonly
+         * @private
```

Comment:
Why bother adding this if it's marked `@private` ?

Reply:
Pretty much just following what `contentBoundingVolume` does. It looks nicer when other files that need this call `tile.boundingVolume` instead of `tile._boundingVolume`. It can't be public though since `TileBoundingVolume` is private.

Reply:
Ah okay.  We want to access it from other classes without adding it to the public API

Reply:
Yup